bookwyrm/templatetags/utilities.py

```diff
@@ -50,3 +51,13 @@ def truncatepath(value, arg):
     except ValueError:  # invalid literal for int()
         return path_list[-1]  # Fail silently.
     return "%s/…%s" % (path_list[0], path_list[-1][-length:])
+
+
+@register.simple_tag(takes_context=False)
+def get_book_cover_thumbnail(book, size, ext):
+    """Returns a book thumbnail at the specified size and extension, with fallback if needed"""
```

Comment:
I'm hitting error when `size` is undefined here (I'm not sure where that's coming from) - I think a reasonable solution would be to use `'medium'` as a default value? Something like `size = size or "medium"`?

Reply:
ah yes, good idea!

Reply:
Would something like this work for you, or do you prefer positional args?

```diff
diff --git a/bookwyrm/templatetags/utilities.py b/bookwyrm/templatetags/utilities.py
index 1be66049..bcb1ec4e 100644
--- a/bookwyrm/templatetags/utilities.py
+++ b/bookwyrm/templatetags/utilities.py
@@ -54,7 +54,7 @@ def truncatepath(value, arg):
 
 
 @register.simple_tag(takes_context=False)
-def get_book_cover_thumbnail(book, size, ext):
+def get_book_cover_thumbnail(book, size="medium", ext="jpg"):
     """Returns a book thumbnail at the specified size and extension, with fallback if needed"""
     try:
         cover_thumbnail = getattr(book, "cover_bw_book_%s_%s" % (size, ext))
```

Reply:
that works! that's better than what I suggested imo